0000-offline-resolution-field.md

```diff
@@ -0,0 +1,73 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Yarn Issue: (leave this empty)
+
+# Summary
+
+When enabling the offline mirror, Yarn updates the lockfile by stripping the registry URL from its `resolved` fields. This RFC aims to simplify this process by making such an update unneeded.
+
+Related issues: https://github.com/yarnpkg/yarn/issues/393 / https://github.com/yarnpkg/yarn/issues/394
+
+# Motivation
+
+Yarn currently has two different type of values for the lockfile `resolved` fields:
+
+  - When online, they're in the form `${source}/${name}-${version}.tar.gz#${hash}`
```

Comment:
also it can be a git URL e.g. ssh://git@bitbucket.org/bestander/test-repo.git, a link: or a file:.